src/ChosenOne.js

```diff
@@ -1,32 +1,44 @@
-import React, { Component} from 'react';
+// import React, { Component} from 'react';
 import './ChosenOne.css';
 
-class ChosenOne extends Component {
-  constructor({details}) {
-    super()
-    this.details = details
-    this.state = {
-      movie: []
-    }
-    console.log(props)
-  }
-  componentDidMount() {
-    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.details.id}`)
-    .then(res => res.json())
-    .then(data => this.setState({movie: data.movie}))
-    .catch(err => console.log(err))
-  }
+const ChosenOne = (props) => {
+
+return (
+  <div 
+    className='single-movie'
+    >
+    <h1>{props.details.title}</h1>
+    <h2>{props.details.overview}</h2>
+    <img className='movie-pic'src={props.details.backdrop_path} alt='movie poster' />
```

Comment:
Can you interpolate the `alt` text here so the movie's title is included? i.e. "poster of <movie title>"